zap/src/main/java/org/parosproxy/paros/view/View.java

```diff
@@ -836,7 +837,7 @@ private ContextGeneralPanel getContextGeneralPanel(Context context) {
         for (AbstractParamPanel panel : contextPanels) {
             if (panel instanceof ContextGeneralPanel) {
                 ContextGeneralPanel contextGeneralPanel = (ContextGeneralPanel) panel;
-                if (contextGeneralPanel.getContextIndex() == context.getIndex()) {
```

Comment:
The method `AbstractContextPropertiesPanel.getContextIndex()` could be changed too.